### PR TITLE
[BUGFIX] AST: Arcana pull states, Sect detection bug

### PR DESCRIPTION
**Pull request by suShirow**
I don't need a changelog for this do I?

### CODE REVIEW

File: src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default class ArcanaTracking extends Module {
 		let prepullSleeve = true
 		const sleeveDrawLog: CastEvent[] = []
 		for (const event of events) {
-			if (event.timestamp - startTime >= (STATUSES.SLEEVE_DRAW.duration * 1000)
+		if (event.timestamp - startTime >= (STATUSES.SLEEVE_DRAW.duration * 1000)
```
Comment (Fank) [inline src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx]: You sure this should be moved?

File: src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx [hunk 2]
```diff
@@ -341,7 +341,8 @@ export default class ArcanaTracking extends Module {
 		const lookupLog = this.cardStateLog.slice(this.pullIndex + 1)
 		if (lookupLog.length > 0) {
 				lookupLog.forEach(cardState => {
```
Comment (Fank) [inline src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx]: Not sure, you didn't changed it. But it looks like there are also some indent issues to

File: src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx (hunk 1 above)
Comment (suShirow) [reply src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx]: O_O uh tslint? Hewwo?? waht r u doin stahp

**Comment by suShirow** [comment]
Okay I'm done here - next update will be a different PR (and hopefully new features instead of bugfixes). lmk if something doesn't check out here

**Pull request merged**